docker/pg_tests/scripts/tests/pgbackrest_backup_fetch_test.sh

```diff
@@ -22,14 +22,18 @@ echo "archive_timeout = 600" >> ${PGDATA}/postgresql.conf
 mkdir -m 770 /tmp/pgbackrest-backups
 
 pgbackrest --stanza=main --pg1-path=${PGDATA} --repo1-path=/tmp/pgbackrest-backups stanza-create
-pgbench -i -s 5 postgres
 
+pgbench -i -s 5 postgres
+pg_dumpall -f /tmp/dump1
+pgbench -c 2 -T 100000000 -S &
```

Comment:
I suggest to use the `pgbench -c 2 -T 1000 &` since the select-only workload doesn't test anything.